moved files to archive and added draft website

AmyMcGann opened the pull request:
I moved Clare's files to archive and added files for the draft website.

shannonmcharg commented:
@AmyMcGann This looks like a great start.

Here are some thoughts:
* Is there any reason not to put the visualizations front and center on the homepage? Maybe as a tabbed component if there is more than one election to show?  Understanding how we got the data feels like it should be a secondary user task to first viewing the data.  
* It feels like Future Work and Recommendations could be consolidated onto a single About page.
* The gray background in the header could be slightly lighter to bring it up to AAA level of accessibility.  Try: #9B9D9D
* The gray background is not accessible with the white text in the footer.  I would either change the text to black or darken it to #4E504E.
* OR you could use the color palette from the OpenMaine website, so all our products start getting a similar look and feel.

AlexPixley commented:
I like the idea of using a common color palette for “branding”.  You only have to solve the majority of accessibility issues once, and it gives anything that uses that color palette a ‘We made this’ feel.

 

From: Shannon McHarg <notifications@github.com> 
Sent: Friday, September 27, 2019 14:19
To: OpenMaine/campaign-finance <campaign-finance@noreply.github.com>
Cc: AlexPixley <alex.pixley.dances.with.sql@gmail.com>; Review requested <review_requested@noreply.github.com>
Subject: Re: [OpenMaine/campaign-finance] moved files to archive and added draft website (#10)

 

@AmyMcGann <https://github.com/AmyMcGann>  This looks like a great start.

Here are some thoughts:

*	Is there any reason not to put the visualizations front and center on the homepage? Maybe as a tabbed component if there is more than one election to show? Understanding how we got the data feels like it should be a secondary user task to first viewing the data.
*	It feels like Future Work and Recommendations could be consolidated onto a single About page.
*	The gray background in the header could be slightly lighter to bring it up to AAA level of accessibility. Try: #9B9D9D
*	The gray background is not accessible with the white text in the footer. I would either change the text to black or darken it to #4E504E.
*	OR you could use the color palette from the OpenMaine website, so all our products start getting a similar look and feel.

—
You are receiving this because your review was requested.
Reply to this email directly, view it on GitHub <https://github.com/OpenMaine/campaign-finance/pull/10?email_source=notifications&email_token=AKVKVDXPVWUYKL5MB657H4LQLZFBHA5CNFSM4I3F5MBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7ZWYIQ#issuecomment-536046626> , or mute the thread <https://github.com/notifications/unsubscribe-auth/AKVKVDWEAQICJANR34T2MFTQLZFBHANCNFSM4I3F5MBA> .  <https://github.com/notifications/beacon/AKVKVDTRAJCIIIVMIDARFADQLZFBHA5CNFSM4I3F5MBKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7ZWYIQ.gif>

AmyMcGann commented:
I changed the colors so they are the same as the OpenMaine website colors.  I also added a visualization to the front page.  I just used one that I had so you can see what it would look like, but would replace it with a Mayoral race visualization when it is ready.
